### PR TITLE
modify includeSelectorTag parameter

### PR DESCRIPTION
**Pull request by morrme**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)
Modified the getHTML file per issue #5603 
## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by morrme** [comment]
@christian-bromann I am not sure what the build issue is, are you familiar with this error?

File: packages/webdriverio/src/commands/element/getHTML.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
  * </example>
  *
  * @alias element.getHTML
- * @param {Boolean=} includeSelectorTag if true it includes the selector element tag (default: true)
+ * @param {Boolean} includeSelectorTag if true it includes the selector element tag (default: true)
```
Comment (christian-bromann) [inline packages/webdriverio/src/commands/element/getHTML.js:25]: I am not sure if this is the right solution to this problem. The `=` at the end of the type marks the parameter as optional we I think we should keep.

Comment (morrme) [reply packages/webdriverio/src/commands/element/getHTML.js:25]: Ah, OK, then let's explore modifying the template as you suggested above. I can change this back. :)

**Comment by christian-bromann** [comment]
@morrme I was running into this issue while working on a set of new commands today and fixed it as part of my PR (see [`96e1daa` (#5477)](https://github.com/webdriverio/webdriverio/pull/5477/commits/96e1daaf9de0675a389e6e921c5f98ea0c610311)). That PR will be merged tomorrow. 

Thanks for looking into it anyway. I would love to support with a different ticket if you are up for it.

**Pull request closed without merging**